lib/veeqo/resources/order.rb

```diff
@@ -54,5 +54,10 @@ def self.destroy(resource_id, params = {})
     rescue Veeqo::NotFound
```

Comment:
As I understand, if body is empty, gem will not catch NotFound 
 error: https://github.com/coaxsoft/veeqo_api_ruby/pull/8/files#diff-d751a380d1e5ae4dc785cfc976ab72e7R53

Reply:
Right now through api you can't destroy an order.
If body is empty, but status is present, then, by that status error is raised, if it's http error.